analyzer/src/test/java/com/novoda/sqlite/impl/SQLFileTest.java

```diff
@@ -0,0 +1,93 @@
+package com.novoda.sqlite.impl;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.List;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class SQLFileTest {
+    private final static String STATEMENT_CREATE_TABLE = "CREATE TABLE banana(_id INTEGER);";
+    private final static String STATEMENT_ALTER_TABLE = "ALTER TABLE banana ADD COLUMN colour TEXT;";
+    private final static String NEW_LINE = "\n";
+    private final static String SPACE = " ";
```

Comment:
Each statement spanning over multiple line will be built on one line, concatenating with a single space. Asserting by manually concatenating space instances (`" "`) could trick the occasional developer into thinking that the space is just "any character".

With regards to extraction, what could be has already been extracted, see `EXPECTED_STATEMENTS`.
`STATEMENT_FOR_COMMENTS_WITHIN_TESTS_1 + SPACE + STATEMENT_FOR_COMMENTS_WITHIN_TESTS_2` remains twice in order to make it explicit that the statement, previously separated by a comment, is now concatenated with a space.